velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -144,9 +144,12 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
   auto frameStart = frameBound(startBoundOptions[startBoundIndex]);
   auto frameEnd = frameBound(endBoundOptions[endBoundIndex]);
 
+  bool isDefaultFrame =
+      (frameStart == "UNBOUNDED PRECEDING" && frameEnd == "CURRENT ROW");
```

Comment:
@JkSelf Comparing strings is error prone. It would be more robust to compare startBoundOptions[startBoundIndex] and endBoundOptions[endBoundIndex] enums.